ca_tools/launch/keyboard_teleop.launch

```diff
@@ -1,3 +1,5 @@
+<!-- USE MOVE_ROBOT VARIABLE TO SELECT WHICH ROBOT TO CONTROL, DEFAULT=1 -->
 <launch>
-  <node pkg="ca_tools" type="key_teleop.py" name="keyboard_teleop" output="screen"/>
+  <arg name="nr" default="$(optenv MOVE_ROBOT 1)"/>
+   <node pkg="ca_tools" type="key_teleop.py" name="keyboard_teleop" output="screen" ns="$(eval 'create' + str(arg('nr')))" if="$(eval arg('nr') > 0)" />
```

Comment:
I _think_ if you create a `xacro:property` and as the value, you put `${lambda x:  x < 0 }` or something like that, you could use functions and reuse the code.